custom_components/solarfocus/sensor.py

```diff
@@ -511,14 +513,16 @@ def native_value(self):
         state_class=SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
-        key="status",
+        key="state",
```

Comment:
🔥  breaking change ... but we'll take it for consistency

Reply:
ups, I've changed it back during testing - I'd keep status to not break it -  must be updated in parallel with pysolarfocus